Fix missing array code convention at Test files

Monsteel opened the pull request:
Bug/issue #, if applicable: N/A

## Summary

Fix missing array code convention at Test files

## Dependencies

N/A

## Testing

No functional changes.

## Checklist

- [x] Added tests
- [x] Ran the `./bin/test` script and it succeeded
- [x] ~Updated documentation if necessary~ N/A

Monsteel commented on Tests/SwiftDocCTests/Infrastructure/DataAssetManagerTests.swift:
```diff
@@ -179,7 +196,9 @@ class DataAssetManagerTests: XCTestCase {
-        let images = ["image.png", "woof~dark.JPG"].compactMap(URL.init(string:))
+        let images = ["image.png",
+                      "woof~dark.JPG",
+        ].compactMap(URL.init(string:))
```
@franklinsch  Thank you for your kind comment. 🙇🏻‍♂️
I've checked all the codes related to your comment, I reflected your comment.

I did my best. ☺️

franklinsch commented on Tests/SwiftDocCTests/Infrastructure/DataAssetManagerTests.swift:
```diff
@@ -179,7 +199,15 @@ class DataAssetManagerTests: XCTestCase {
+=======
+          "image.png",
+          "woof~dark.JPG",
+>>>>>>> Stashed changes
```
It looks like there is a conflict here

Monsteel replied:
I'm sorry, but I couldn't check it. 😓

franklinsch commented on Tests/SwiftDocCTests/Infrastructure/DataAssetManagerTests.swift:
```diff
@@ -196,12 +224,19 @@ class DataAssetManagerTests: XCTestCase {
         // Test we include only unique keys in the fuzzy index
-        let imagesWithDuplicates = ["image@2x.png", "assets/woof.JPG", "woof.png"].compactMap(URL.init(string:))
+        let imagesWithDuplicates = [
+          "image@2x.png",
```
To be consistent with the rest of the code, we should use a 4-space indent for these

Monsteel replied:
Thank you for your kind comment. ☺️

franklinsch commented on Tests/SwiftDocCTests/Infrastructure/DocumentationContextTests.swift:
```diff
@@ -553,8 +553,24 @@ class DocumentationContextTests: XCTestCase {
-    }
+        XCTAssertEqual(
+          [
+            "figure1.jpg",
```
This one is still using a two-space indent. I'd expect 4 spaces here for consistency

franklinsch commented on Tests/SwiftDocCTests/Infrastructure/SymbolReferenceTests.swift:
```diff
@@ -211,9 +211,11 @@ class SymbolReferenceTests: XCTestCase {
-             "/documentation/OverloadKit/function(_:)-4s9wl",
-             "/documentation/OverloadKit/function(_:)-7kt4g"]
+            [
+              "/documentation/OverloadKit/function(_:)-1echv",
```
This one as well

franklinsch reviewed (commented):
Thanks @Monsteel ! It looks like there are still some cases where we're using two-space indents. Since the goal of this PR is to introduce consistency, I think we should fix those up before merging :)

Monsteel commented:
@franklinsch I'm sorry I couldn't look at all the files with interest.
Among the files I changed, I checked and corrected the incorrect part of space indent convention.

If there's anything I missed,
Please let me know. I'll revise it.

Thank you for your kind confirmation.

Monsteel commented:
@franklinsch Please check..!😓

ethan-kusters commented:
Hi @Monsteel! This looks good-to-go from my perspective. Thank you for your work here!

If you could resolve the remaining conflicts, I'll re-run CI and merge.

Monsteel commented:
Hello @ethan-kusters ! Thank you so much. 🤗
I solved the remaining conflicts.🙋🏻‍♂️ Please check. 🙏

ethan-kusters commented:
@swift-ci please test

ethan-kusters commented:
@swift-ci please test

Monsteel commented:
@ethan-kusters Thanks, But I don't understand swiftTest result. please tell me about test result ?😓

ethan-kusters commented:
@Monsteel could you manually rebase on `main` and force-push your changes? It looks like just merging main in the GitHub UI is not picking up the latest changes.

Monsteel commented:
@ethan-kusters I did manually rebase on main and force-push my changes🙋🏻‍♂️ Thanks for your hard work! 🙇🏻‍♂️

ethan-kusters commented:
@swift-ci please test

ethan-kusters reviewed (approved):
Thanks @Monsteel!

Monsteel commented:
Thank you, @ethan-kusters, @franklinsch !

Through this opportunity, **I felt the kindness of Apple developer,  I experienced small collaboration.** so I'm happy.🤩
I hope you have a happy coding. 🙏

Thanks!
